config.json.dist

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
`config.json.dist` and `secrets.json.dist` have the same content.

Perhaps this file will contain constants like `CWRKNG_ENDPOINT` and `CWRKNG_DOOR`?